pkg/sql/lease.go

```diff
@@ -761,6 +831,10 @@ func (t *tableState) removeInactiveVersions(m *LeaseManager) {
 func (t *tableState) acquireWait() {
 	// Spin until no lease acquisition is in progress.
 	for acquiring := t.mu.acquiring; acquiring != nil; acquiring = t.mu.acquiring {
+		// Indicates that a routine has blocked at all on another routine acquiring.
+		// Used for testing.
+		t.mu.hasBlockedDuringAcquisition = true
+
```

Comment:
You probably need a test that runs a ton of acquisitions and ensures that they don't get blocked.